tests/test_read.py

```diff
@@ -377,3 +377,28 @@ def test_index_unit_equals_m():
 def test_read_incorrect_shape():
     with pytest.raises(ValueError):
         lasio.read(egfn("sample_lastcolblanked.las"))
+
+def test_dot_delmiter_issue_264():
```

Comment:
Could you correct the spelling mistake in the function name?

Reply:
Yep, Fixing.  Thanks for catching the error.

DC